flash_attn/flash_attn_interface.py

```diff
@@ -45,11 +46,57 @@ def _get_block_size_n(device, head_dim, is_dropout, is_causal):
         return 64
 
 
+def round_multiple(x, m):
+    return (x + m - 1) // m * m
+
+
+def _compile_wrapper(torch_op, func):
+    if torch.__version__ >= "2.4.0":
+        return torch_op
+    else:
+        return func
+
+
+if torch.__version__ >= "2.4.0":
+    _torch_custom_op_wrapper = torch.library.custom_op
+    _torch_register_fake_wrapper = torch.library.register_fake
+else:
+    def custom_op_wrapper(name, fn=None, /, *, mutates_args, device_types=None, schema=None):
```

Comment:
Nit - a more desciptive name - noop_custom_op_wrapper

And maybe a comment that we dont support < 2.4 versions.